### PR TITLE
Remove superflous property and adding get all defaults behaviorgroups

### PR DESCRIPTION
**Pull request by josejulio**
- It doesn't make sense to specify a groupId in the default email action
   as is not linked to any account
 - Adds endpoint to query all the default behavior groups
 - Adding tests

### CODE REVIEW

File: backend/src/test/java/com/redhat/cloud/notifications/routers/InternalServiceTest.java [hunk 1]
```diff
@@ -251,6 +258,49 @@ void testCreateAndGetAndUpdateAndDeleteEventType() {
         getEventTypes(appId, OK, 1);
     }
 
+    @Test
+    void testDefaultBehaviorGroupCRUD() {
+        // We need to persist a bundle, an app and two event types for this test.
```
Comment (gwenneg) [inline backend/src/test/java/com/redhat/cloud/notifications/routers/InternalServiceTest.java:263]: Where are the app and the event types?

File: backend/src/test/java/com/redhat/cloud/notifications/routers/InternalServiceTest.java [hunk 2]
```diff
@@ -570,4 +620,77 @@ private static void deleteEventType(String eventTypeId, boolean expectedResult)
...
+
+    private static void updateDefaultBehaviorGroup(String behaviorGroupId, String displayName, String bundleId, boolean expectedResult) {
+        BehaviorGroup behaviorGroup = buildDefaultBehaviorGroup(displayName, bundleId);
+
+        Boolean result = given()
+                .basePath(API_INTERNAL)
+                .contentType(JSON)
+                .body(Json.encode(behaviorGroup))
+                .pathParam("behavior_group_id", behaviorGroupId)
+                .put("/behaviorGroups/default/{behavior_group_id}")
```
Comment (gwenneg) [inline backend/src/test/java/com/redhat/cloud/notifications/routers/InternalServiceTest.java:656]: We could use a consistent path between runtime and test to make searches easier:
```suggestion
                .pathParam("id", behaviorGroupId)
                .put("/behaviorGroups/default/{id}")
```

File: backend/src/test/java/com/redhat/cloud/notifications/routers/InternalServiceTest.java [hunk 3]
```diff
@@ -570,4 +620,77 @@ private static void deleteEventType(String eventTypeId, boolean expectedResult)
...
+                .extract().as(Boolean.class);
+
+        assertEquals(expectedResult, result);
+    }
+
+    private static void deleteDefaultBehaviorGroup(String behaviorGroupId, boolean expectedResult) {
+        Boolean result = given()
+                .basePath(API_INTERNAL)
+                .pathParam("behavior_group_id", behaviorGroupId)
+                .delete("/behaviorGroups/default/{behavior_group_id}")
```
Comment (gwenneg) [inline backend/src/test/java/com/redhat/cloud/notifications/routers/InternalServiceTest.java:668]:
```suggestion
                .pathParam("id", behaviorGroupId)
                .delete("/behaviorGroups/default/{id}")
```

File: backend/src/test/java/com/redhat/cloud/notifications/routers/InternalServiceTest.java (hunk 1 above)
Comment (josejulio) [reply backend/src/test/java/com/redhat/cloud/notifications/routers/InternalServiceTest.java:263]:
```suggestion
        // We need to persist a bundle.
```

Didn't use them in the end and forgot to update :D

**Review by gwenneg — APPROVED** [review]
LGTM

**Pull request merged**